AsyncDisplayKit/ASButtonNode.mm

```diff
@@ -126,6 +140,25 @@ - (void)setLaysOutHorizontally:(BOOL)laysOutHorizontally
   [self setNeedsLayout];
 }
 
+
+- (void)setTitle:(NSString *)title forState:(ASButtonState)state
+{
+  UIColor *currentTitleColor = [self titleColorForState:state];
+  NSDictionary *attributes = @{
+                               NSFontAttributeName: [UIFont systemFontOfSize:[UIFont buttonFontSize]],
```

Comment:
I like the idea of the simplified setTitle:, but I'm afraid that very few apps will want to just use the off-the-shelf buttonFontSize.  I don't think the API as it stands adds enough convenience with sufficient expressive power, at least to justify the number of added methods.

I wonder if a setTitle:withFont:color:forState: would be a solution.  Not sure those are the right parameter names / grammar for the method, but it seems closer to optimal?